ocs_ci/deployment/vmware.py

```diff
@@ -371,6 +371,18 @@ def __init__(self):
                 'installer'
             )
             self.previous_dir = os.getcwd()
+
+            # Download terraform binary based on ocp version and
+            # update in ENV_DATA
+            # use "0.11.14" for releases below OCP 4.5
+            ocp_version = get_ocp_version()
+            terraform_version = constants.TERRAFORM_OLD_VERSION
+            if Version.coerce(ocp_version) >= Version.coerce('4.5'):
+                terraform_version = config.DEPLOYMENT['terraform_version']
```

Comment:
if you will go with default_terraform_version you can do this here
```suggestion
           terraform_version = config.DEPLOYMENT.get('terraform_version', constants.TERRAFORM_OLD_VERSION)
```

Reply:
As discusses, chnaged on all yaml files